docker/README.md

```diff
@@ -0,0 +1,30 @@
+# Docker image for sendgrid-ruby
+
+## Quickstart
+1. [Install Docker](https://docs.docker.com/engine/installation/) on your machine.
+2. Run `docker run --rm -it sendgrid/sendgrid-ruby irb`.
+3. Run `require './lib/sendgrid-ruby.rb'`.
+
+## Poke around
+
+If you would like to just poke around in the image and check some examples:
+```sh
+docker run --rm -it sendgrid/sendgrid-ruby bash
+```
+
+If you want to mount your fork or specific version of the gem:
+```sh
+docker run --rm -v /path/to/local/sendgrid-ruby:/sources/sendgrid-ruby -it sendgrid/sendgrid-ruby bash
```

Comment:
Would you be up for adding a docker-compose file that would make the ruby gems and source code local to the machine the container is running on?

Reply:
Will the volumes not be sufficient for this?